homeassistant/components/izone/climate.py

```diff
@@ -562,68 +582,75 @@ def target_temperature_step(self) -> float:
         return 0.5
 
     @property
+    @_return_on_connection_error(0.0)
     def min_temp(self) -> float:
         """Return the minimum temperature."""
         return self._controller.min_temp
 
     @property
+    @_return_on_connection_error(50.0)
     def max_temp(self) -> float:
         """Return the maximum temperature."""
         return self._controller.max_temp
 
     @property
-    def airflow_min(self):
+    @_return_on_connection_error(0)
+    def airflow_min(self) -> int:
         """Return the minimum air flow."""
         return self._zone.airflow_min
 
     @property
-    def airflow_max(self):
+    @_return_on_connection_error(100)
+    def airflow_max(self) -> int:
         """Return the maximum air flow."""
         return self._zone.airflow_max
 
+    @_reraise_on_connection_error()
     async def async_set_airflow_min(self, **kwargs):
         """Set new airflow minimum."""
-        await self._controller.wrap_and_catch(
-            self._zone.set_airflow_min(int(kwargs[ATTR_AIRFLOW]))
-        )
+        await self._zone.set_airflow_min(int(kwargs[ATTR_AIRFLOW]))
         self.async_write_ha_state()
 
+    @_reraise_on_connection_error()
     async def async_set_airflow_max(self, **kwargs):
         """Set new airflow maximum."""
-        await self._controller.wrap_and_catch(
-            self._zone.set_airflow_max(int(kwargs[ATTR_AIRFLOW]))
-        )
+        await self._zone.set_airflow_max(int(kwargs[ATTR_AIRFLOW]))
         self.async_write_ha_state()
 
+    @_reraise_on_connection_error()
     async def async_set_temperature(self, **kwargs: Any) -> None:
         """Set new target temperature."""
         if self._zone.mode != Zone.Mode.AUTO:
             return
         if (temp := kwargs.get(ATTR_TEMPERATURE)) is not None:
-            await self._controller.wrap_and_catch(self._zone.set_temp_setpoint(temp))
+            await self._zone.set_temp_setpoint(temp)
 
+    @_reraise_on_connection_error()
     async def async_set_hvac_mode(self, hvac_mode: HVACMode) -> None:
         """Set new target operation mode."""
         mode = self._state_to_pizone[hvac_mode]
-        await self._controller.wrap_and_catch(self._zone.set_mode(mode))
+        await self._zone.set_mode(mode)
         self.async_write_ha_state()
 
     @property
+    @_return_on_connection_error(False)
```

Comment:
If the `available` property returns False, this property will never be requested.